flamedisx/xenon/x1t_sr1.py

```diff
@@ -305,10 +305,16 @@ def electron_gain_std(s2_relative_ly,
         # 0 * light yield is to fix the shape
         return single_electron_width + 0. * s2_relative_ly
 
+    @staticmethod
+    def double_pe_fraction(z, *, dpe=DEFAULT_P_DPE):
+        # Ties the double_pe_fraction model function to the dpe
+        # parameter in the sources
+        return dpe + 0 * z
```

Comment:
Should be `0. * z`

Reply:
This already works because of the typecast of z. But good to have it explicit.